explorer_pb/explorer_server.go

```diff
@@ -0,0 +1,40 @@
+package explorer_pb
```

Comment:
this file could be moved to "server" directory

Reply:
Done, I also initially thought of it.